cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -80,32 +88,35 @@
     private static final String LOG_TAG = "BasicDatastore";
     private static final Logger logger = Logger.getLogger(DatastoreImpl.class.getCanonicalName());
 
-    private static final String FULL_DOCUMENT_COLS = "docs.docid, docs.doc_id, revid, sequence, json, current, deleted, parent";
+    public static final String FULL_DOCUMENT_COLS = "docs.docid, docs.doc_id, revid, sequence, " +
+            "json, current, deleted, parent";
 
-    private static final String GET_DOC_NUMERIC_ID =
+    public static final String GET_DOC_NUMERIC_ID =
```

Comment:
Do you think there is any value in moving these into the `SQLCallable`s that use them? IIRC most are used only once. If they are more widely shared a better option might be putting them in a separate constants class. I guess if we wanted them more tightly bound to the callables they could be added to the `SQLCallable` interface or an abstract super. I just think it is weird keeping the SQL statements here now that this class basically needs to know nothing about SQL.